### PR TITLE
Updated LSF function

### PR DESCRIPTION
**Pull request by Lucrezia-Cester**

### CODE REVIEW

File: tests/test_spatial_resolution.py [hunk 1]
```diff
@@ -150,9 +153,10 @@ class TestPhilipsResolution(TestSpatialResolution):
     EDGE_MEAN = 133.57
     TOP_EDGE_MEAN = 205.28
     SIGNAL_MEAN = 348.5525
-    MTF_FE = 0.4923415061063675
+    MTF_FE =  0.6017507296855603
     MTF_PE = 0.4923415061063675
     bisecting_normal = (281, 245, 319, 239)
+    print('this is phil',hazen_spatial_resolution.calculate_mtf(dicom))
```
Comment (tomaroberts) [inline tests/test_spatial_resolution.py:159]: Did you mean to leave this in?

Comment (tomaroberts) [inline tests/test_spatial_resolution.py]: Did you mean to leave this in?

File: tests/test_spatial_resolution.py [hunk 2]
```diff
@@ -170,3 +174,4 @@ class TestEastKentResolution(TestSpatialResolution):
     MTF_FE = 0.9924200730536655
     MTF_PE = 0.9924200730536658
     bisecting_normal = (137, 126, 155, 122)
+    print('this is east',hazen_spatial_resolution.calculate_mtf(dicom))
```
Comment (tomaroberts) [inline tests/test_spatial_resolution.py]: And this one?

**Review by tomaroberts — CHANGES_REQUESTED** [review]
Couple of minor comments. 

Would you mind adding a description to the initial PR post as well for future reference? And link any issues.

**Comment by Lucrezia-Cester** [comment]
@tomaroberts I changed according to the comments.

**Pull request merged**